msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -56,11 +56,13 @@ Teams automatically crops your icon to display a square with rounded corners in
 
 An outline icon displays in two scenarios:
 
-* When your app is in use and “hoisted” on the app bar on the left side of Teams.
+* When your app is in use and pinned to the app bar on the left side of Teams.
```

Comment:
Good call--I haven't encountered the term "hoisted" anywhere else in the docs, whereas "pinned" seems standard

Reply:
if "pinned" is the consistent term that's fine, but I would suggest something like "visible on the app bar" as the more appropriate umbrella term - apps are visible on the app bar when they are _either_ pinned or hoisted.